examples/templates/cluster-operator/mirror-maker-template.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: strimzi-mirror-maker
+  annotations:
+    openshift.io/display-name: "Apache Kafka MirrorMaker"
+    description: >-
+      This template installs Kafka MirrorMaker to mirror Kafka clusters. 
+      For more information about using this template see http://strimzi.io
+    tags: "messaging,datastore"
+    iconClass: "fa fa-cubes"
+    template.openshift.io/documentation-url: "http://strimzi.io"
+message: "Kafka MirrorMaker is being deployed"
+parameters:
+- description: Bootstrap server of the source cluster to be mirrored
+  displayName: Bootstrap server of the source cluster
+  name: MIRROR_MAKER_SOURCE
+  required: true
+  value: "my-source-cluster-kafka-bootstrap:9092"
+- description: Bootstrap server of the target cluster to mirror source cluster
+  displayName: Bootstrap server of the target cluster
+  name: MIRROR_MAKER_TARGET
+  required: true
+  value: "my-target-cluster-kafka-bootstrap:9092"
+- description: Number of MirrorMaker cluster nodes which will be deployed
+  displayName: Number of MirrorMaker cluster nodes
+  name: MIRROR_MAKER_NODE_COUNT
+  required: true
+  value: "1"
+- description: The group of consumer processes to which this consumer belongs
```

Comment:
I think you should use the term "consumer group id", because people will immediately recognise that, whereas I'm not so sure they'd realise what "group of consumer processes" meant.